firmware/application/apps/ui_recon.cpp

```diff
@@ -992,6 +1023,10 @@ void ReconView::frequency_file_load(bool stop_all_before) {
 }
 
 void ReconView::on_statistics_update(const ChannelStatistics& statistics) {
+    chrono_end = chTimeNow();
```

Comment:
What's the resolution on this clock, do you know? I need a fast clock for the auto step encoder thingy.

Reply:
It's milliseconds when it can, but I'ù not sure of the granularity if you time too quick